### DISCUSSION TITLE
Download resources

### DISCUSSION THREAD

**Discussion by trevorsaint** [ONSdigital/design-system#888] [Feedback]
Use this place to discuss the [download resources pattern](https://ons-design-system.netlify.com/patterns/download-resources) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by armstrongb** [comment]
Preview of this in the design system has a page of extreme length and it slowly auto-scrolls upwards back into view. (The scrolling back up in the gif below is not user controlled.)

![320-design-system-download-resources-pattern-scroll](https://user-images.githubusercontent.com/22910336/95750652-5cb5a280-0c95-11eb-8779-1d8a28dda19f.gif)

**Reply by trevorsaint** [reply]
This has been fixed @armstrongb

**Comment by trevorsaint** [comment]
@armstrongb I noticed this also. I thought it was my computer playing funny buggers.

I would suspect this has something to do with the iframe resize, and not the component itself.

**Comment by jrbarnes9** [comment]
Downloadable resources placeholder thumbnail for census is in English only

**Comment by jrbarnes9** [comment]
Download filter labels overlapping grid col
![Screenshot 2021-03-08 at 09 16 52](https://user-images.githubusercontent.com/43346934/110300736-246d8280-7fef-11eb-8008-40c11e693e29.png)

**Comment by jrbarnes9** [comment]
Console error in download resources pattern when selecting filters

<img width="1275" alt="Screenshot 2021-10-11 at 12 45 05" src="https://user-images.githubusercontent.com/43346934/136784853-fe132028-afc5-49b0-8204-eb36dae03139.png">

**Reply by jrbarnes9** [reply]
Raised #1664

**Comment by jrbarnes9** [comment]
Need to either hide filters when no js, or add a submit button to filters and build a server-side solution.

**Reply by jrbarnes9** [reply]
Raised #2027

**Comment by jrbarnes9** [comment]
Does this pattern require pagination? If so, which type? Lazy loading, 'load more' link, pager?

**Reply by jrbarnes9** [reply]
Raised #2030
